controllers/app.controllers.js

```diff
@@ -23,15 +35,16 @@ exports.getArticles = (req, res, next) =>{
     })
 }
 
-exports.getArticleById = (req, res, next) => {
+exports.getCommentsByArtId = (req, res, next) =>{
     const {article_id} = req.params
     if (isNaN(article_id)) return next({ status: 400, message: 'Not a number, please enter valid id'});
-
-    fetchArticleById(article_id)
-    .then((article)=>{
-        res.status(200).send({ article })
+    fetchArticleById(article_id).then(()=>{
+        fetchCommentsByArtId(article_id)
```

Comment:
Please eliminate the nesting of .then()'s here.

Reply:
done